src/locations/mcda_criteria.jl

```diff
@@ -0,0 +1,268 @@
+using ADRIA: mcda_normalize
```

Comment:
At a high level, I'm struggling to understand conceptually why lump "locations" with "mcda".
Can you elaborate why MCDA related files are grouped under the "locations" directory?

If I were cleaning it up, I think I'd consider separating MCDA out into its own under some other label/name.

Reply:
I simply changed the sites file to "locations" which originally had the mcda stuff in it, but this can be changed.